cider/tests/utils/CiderTestBase.cpp

```diff
@@ -48,15 +48,17 @@ void CiderTestBase::prepareArrowBatch() {
 }
 
 void CiderTestBase::assertQueryArrow(const std::string& sql,
-                                     const std::string& json_file) {
+                                     const std::string& json_file,
```

Comment:
I remember there is a method called `assertQueryArrowIgnoreOrder`, this one is duplicated.

Reply:
So I think we also need a method called `assertQueryArrowOrdered` or we could just keep the same as before.

Reply:
Previous we use the flag to determine ignore order or not. But it's also okay to use `assertQueryArrow` and `assertQueryArrowIgnoreOrdered`, which means that by default it's ordered compared. At least duplication is not allowed.

Reply:
Changed to `assertQueryArrowIgnoreOrder` and removed.